web/.env.example

```diff
@@ -6,8 +6,8 @@ SECRET_KEY="create a strong key"
 
 # Used for Authentication
 # More info: https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app
-GITHUB_APP_CLIENT_ID=""
-GITHUB_APP_CLIENT_SECRET=""
+GH_APP_CLIENT_ID=""
+GH_APP_CLIENT_SECRET=""
```

Comment:
**dev note**: This was changed from `GITHUB_` to `GH_` because Github won't allow us to create a secret that starts with `GITHUB_`.